<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatistikModel</class>
 <widget class="QWidget" name="StatistikModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistik</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>429</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Select customers that have email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="emailComboBox">
      <item>
       <property name="text">
        <string>gmail.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rambler.ru</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mail.ru</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="emailButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>646</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Select customers payments that payed between dates:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateStartEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateFinishEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dateButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>327</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="groupByLabel">
      <property name="text">
       <string>Select number of workers on works:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="worksCountButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>Select number of workers on work:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="workComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="workCountButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="anyLabel">
      <property name="text">
       <string>Is all work status true:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="allButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>353</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>What works were not paid immediately:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="immediatelyButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>319</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="unionLabel">
      <property name="text">
       <string>Show customer and worker table:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unionButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>599</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="updateLabel">
      <property name="text">
       <string>Update the status of work to True but if it was performed on the worker's pay:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="workIdComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
